receiver/elasticsearchreceiver/metadata.yaml

```diff
@@ -483,9 +499,228 @@ metrics:
   elasticsearch.os.memory:
     description: Amount of physical memory.
     unit: By
+    gauge:
+      value_type: int
+    attributes: [memory_state]
+    enabled: true
+  elasticsearch.indexing_pressure.memory.coordinating:
+    description: Memory consumed, in bytes, by indexing requests in the coordinating stage.
+    unit: By
+    gauge:
+      value_type: int
+    attributes: []
+    enabled: true
+  elasticsearch.indexing_pressure.memory.primary:
+    description: Memory consumed, in bytes, by indexing requests in the primary stage.
+    unit: By
+    gauge:
+      value_type: int
+    attributes: []
+    enabled: true
+  elasticsearch.indexing_pressure.memory.replica:
+    description: Memory consumed, in bytes, by indexing requests in the primary stage.
+    unit: By
+    gauge:
+      value_type: int
+    attributes: []
+    enabled: true
+  elasticsearch.indexing_pressure.memory.total.primary_rejections:
+    description: Cumulative number of indexing requests rejected in the primary stage.
+    unit: 1
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [ ]
+    enabled: true
+  elasticsearch.indexing_pressure.memory.total.replica_rejections:
+    description: Number of indexing requests rejected in the replica stage.
+    unit: 1
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [ ]
+    enabled: true
+  elasticsearch.indexing_pressure.memory.limit:
+    description: Configured memory limit, in bytes, for the indexing requests.
+    unit: By
+    gauge:
+      value_type: int
+    attributes: [ ]
+    enabled: true
+  elasticsearch.node.cluster.state_queue.pending:
```

Comment:
`node.cluster` doesn't appear to be a hierarchical namespace. The node belongs to the cluster. Is the node even relevant here, as opposed to just calling this a cluster metric?